pylmod/tests/test_membership.py

```diff
@@ -2,13 +2,151 @@
 """
 Test the membership class
 """
+import httpretty
+import json
+
+
 from pylmod import Membership
 from pylmod.tests.common import BaseTest
 
 
-class TestGradebook(BaseTest):
+class TestMembership(BaseTest):
     """Validate defined methods and constructors in Membership class"""
 
+    # Unit tests generally should do protected-accesses
+    # pylint: disable=protected-access
+    COURSE_ID = 12345
+
+    STAFF_BODY = [
+        {
+            u'displayName': u'Huey Duck',
+            u'role': u'TA',
+            u'sortableDisplayName': u'Duck, Huey'
+        },
+        {
+            u'displayName': u'Louie Duck',
+            u'role': u'CourseAdmin',
+            u'sortableDisplayName': u'Duck, Louie'
+        },
+        {
+            u'displayName': u'Benjamin Franklin',
+            u'role': u'CourseAdmin',
+            u'sortableDisplayName': u'Franklin, Benjamin'
+        },
+        {
+            u'displayName': u'George Washington',
+            u'role': u'Instructor',
+            u'sortableDisplayName': u'Washington, George'
+        },
+    ]
+
+    COURSE_DATA = {
+        u'response':
+        {
+            u'docs':
+            [
+                {
+                    u'googleEnabled': True,
+                    u'groupTemplate': u'project',
+                    u'groupingScheme': None,
+                    u'id': 12345,
+                    u'longName': u'pylmod test Site',
+                    u'name': u'pylmod test Site',
+                    u'properties':
+                    {
+                        u'allowsStudentsToSwitchSection': u'1',
+                        u'lastDateToSwitchSection': u'1480723199000'
+                    },
+                    u'termCode': u'2033FA',
+                    u'uuid': u'/project/testingstuff'
+                }
+            ],
+            u'numFound': 1,
+            u'start': 0
+        },
+        u'responseHeader':
+            {
+                u'QTime': 0,
+                u'params':
+                {
+                    u'wt': u'json'
+                },
+                u'qtime': 0,
+                u'status': 0
+            }
+    }
+
+    def _register_get_membership(self, send_data=True):
```

Comment:
this isn't being called in tests, does this even need to be here? Also, it has a print statement in it.